lib/build.sh

```diff
@@ -105,3 +105,14 @@ run_build() {
     npm run build
   fi
 }
+
+write_launch_toml() {
+  local build_dir=$1
```

Comment:
why are you passing this in?

Reply:
Good catch. I was going to check for `Procfile` but realized I don't need to. I wonder I should check if that key actually exists.

Reply:
Just following up from our Slack convo... we should check to see if there's an `npm start` in case they haven't specified an `npm start` script

Reply:
(`jq` is available for parsing `package.json` too)